package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Testing framwork for @hmcts/one-per-page",
   "homepage": "https://github.com/hmcts/one-per-page-test-suite#readme",
   "main": "./index.js",
-  "version": "2.0.17",
+  "version": "2.0.18",
```

Comment:
Should this be 2.1.0 to reflect that we upgraded a 1.x.x dependency?

Reply:
Boris pointed out to me that we use ^ for getting versions in the other repos.
Meaning only patch is auto updated it seems. I don't think this update of look-and-feel significantly changes things, and would be useful for all out apps to automatically pull I guess?
In this case lets leave as patch?

Reply:
Just double checked and its fine either way according to:
https://devhints.io/semver
But either way is fine I'd say.